helpers/networker.py

```diff
@@ -0,0 +1,85 @@
+from helpers import remote, sysnet, tf_cfg
+
+
+class NetWorker:
+    def _get_state(self, dev, what):
+        cmd = f"ethtool --show-features {dev} | grep {what}"
+        out = remote.client.run_cmd(cmd)
+        return out[0].decode("utf-8").split(" ")[-1].strip("\n")
+
+    def _set_state(self, dev, what, on=True):
+        if on:
+            cmd = f"ethtool -K {dev} {what} on"
+        else:
+            cmd = f"ethtool -K {dev} {what} off"
+        out = remote.client.run_cmd(cmd)
```

Comment:
There is such logic in `tls/test_tls_integrity.py`, which should be replaced with the new call to NetWorker

Reply:
Fixed